idunn/api/geocoder.py

```diff
@@ -49,7 +51,11 @@ async def get_intentions():
         if query.lon and query.lat:
             focus = Point(query.lon, query.lat)
 
-        return await nlu_client.get_intentions(text=query.q, lang=query.lang, focus=focus)
+        try:
+            return await nlu_client.get_intentions(text=query.q, lang=query.lang, focus=focus)
+        except NluClientException as exp:
+            logger.warning("Ignored NLU for '%s': %s", query.q, exp.reason, extra=exp.extra)
```

Comment:
It would be useful to set `reason` in the `extra` object to make logs filtering easier.

Also, I would use `logger.info` instead of `logger.warning` here as this log message will be used for a large portion of requests, and does not draw attention about a specific problem.